src/selectors/entities/cloud.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSelector} from 'reselect';
+
+import {Invoice} from 'types/cloud';
+import {GlobalState} from 'types/store';
+import {Dictionary} from 'types/utilities';
+
+export function makeGetLastInvoice(): (state: GlobalState) => Invoice | undefined {
```

Comment:
I believe that part of the changes was to have the `Subscription` struct have a field for `LastInvoice` the same way it is modeled in Stripe so we wouldn't need to calculate this. @ethervoid could you confirm if this is something you are adding?

Reply:
Yes, I'm adding that as Ali suggested in the Subscription model as you can see [here](https://github.com/mattermost/customer-web-server/pull/257/files#diff-23410cc4a3f0578104eca2027b742790534897f8c5e79856678663e295f2736fR24)

Reply:
Cool, I'll remove this selector then and add that field in the `Subscription` front-end model